app/helpers.rb

```diff
@@ -23,39 +23,29 @@ def render_value(key, value, name)
 					end.join
 					}</fieldset>"
 				when Array
-					"<fieldset#{" class='hidden'" if key.to_s.start_with?("_")}><legend>#{key}</legend><ol data-key='#{name}'>#{
-						value.each_with_index.map { |v, i| "<li>#{render_value(nil, v, "#{name}[#{i}]")}</li>" }.join
-					}</ol></fieldset>"
+					if multiple_select?(key)
```

Comment:
why put this condition in a helper?  others aren't, and we don't have this is more than one place, do we?